File: client/src/app/_models/Message.ts

export interface Message {
    id: Number;
    senderId: Number;
    senderUserName:string;
    senderPhotoUrl:string;
    recipientId:number;
    recipientUserName:string;
    recipientPhotoUrl:string;
    content:string;
    dateRead:Date;
    messageSent:Date ;


}

File: API/entities/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.entities
{
    public class Message
    {
        public int id { get; set; }
        public int senderId { get; set; }
        public string senderUserName { get; set; }
        public AppUser sender { get; set; }




        public int recipientId { get; set; }
        public string recipientUserName { get; set; }
        public AppUser recipient { get; set; }




        public string content { get; set; }
        public DateTime? dateRead { get; set; }
        public DateTime messageSent { get; set; }=DateTime.UtcNow;





        public bool senderDeleted{get; set;}
        public bool recipientDeleted { get; set; }
    }
}